<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBookDialog</class>
 <widget class="QDialog" name="WBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/bookcase.png</normaloff>:/icons/bookcase.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>cout</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <widget class="QWidget" name="page_wordlist">
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget_wordlist">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="dragDropOverwriteMode">
                <bool>false</bool>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::DragOnly</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>word</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>star</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>time</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_test"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_Remove">
   <property name="text">
    <string>Remove Word List</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>New Word List</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>Export Word List</string>
   </property>
   <property name="toolTip">
    <string>Export Word List</string>
   </property>
  </action>
  <action name="action_RemoveWord">
   <property name="text">
    <string>Remove Word from List</string>
   </property>
  </action>
  <action name="action_StarWordTo">
   <property name="text">
    <string>Star Word to</string>
   </property>
   <property name="toolTip">
    <string>Star Word to</string>
   </property>
  </action>
  <action name="action_Star1">
   <property name="text">
    <string>Star to 1</string>
   </property>
  </action>
  <action name="action_Star2">
   <property name="text">
    <string>Star to 2</string>
   </property>
  </action>
  <action name="action_Star3">
   <property name="text">
    <string>Star to 3</string>
   </property>
  </action>
  <action name="action_Star4">
   <property name="text">
    <string>Star to 4</string>
   </property>
  </action>
  <action name="action_Star5">
   <property name="text">
    <string>Star to 5</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WBookDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WBookDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
